File: practice-challenges/get-words-starting-with-vowels/get_vowel_words.py
"""Solution for Get Words Starting with Vowels"""


def get_words_starting_with_vowels(words):
    vowel_words = []

    for word in words:
        if word[0] in "aeiou":
            vowel_words.append(word)

    return vowel_words


# Tests

# ["elephant", "hello", "octopus"] => ['elephant', 'octopus']
print(get_words_starting_with_vowels(["elephant", "hello", "octopus"]))

# ["hi", "meow", "yay"] => []
print(get_words_starting_with_vowels(["hi", "meow", "yay"]))

File: practice-challenges/get-words-starting-with-vowels/getVowelWords.js
/* Solution for Get Words Starting with Vowels */

function getWordsStartingWithVowels(words) {
  let vowelWords = [];

  for (let word of words) {
    if ("aeiou".includes(word[0])) {
      vowelWords.push(word);
    }
  }

  return vowelWords;
}

// Tests

// ["elephant", "hello", "octopus"] => ['elephant', 'octopus']
console.log(getWordsStartingWithVowels(["elephant", "hello", "octopus"]));

// ["hi", "meow", "yay"] => []
console.log(getWordsStartingWithVowels(["hi", "meow", "yay"]));